README.md

```diff
@@ -143,6 +144,45 @@ This has the effect of adding the following to the `pg_hba.conf` file:
 host    all             all             samenet                 trust
 ```
 
+# Creating a Snapshot or Slave Database
+
+You may use the `PSQL_SLAVE` variable along with `REP_HOST`, `REP_PORT`, `REP_USER` and `REP_PASS` to create a snapshot of an existing database and enable stream replication.
+
+Your master database must support replication or super-user access for the credentials you specify. The `PSQL_SLAVE` variable should be set to "true" for streaming replication or "snapshot" for a simple snapshot of a running database.
+
+Create a new master instance:
+
+```bash
+docker run --name postgresql-master -p 5432:5432 -d \
+  -e 'PSQL_TRUST_LOCALNET=true' \
+  -e 'DB_NAME=dbname' \
+  -e 'DB_USER=dbuser' -e 'DB_PASS=dbpass' \
+  -e 'REP_USER=replicator'  -e 'REP_PASS=replicatorpass' \
+  sameersbn/postgresql:9.4
+```
+
+Create a slave instance with streaming replication:
+
+```bash
+docker run --name postgresql-slave -p 5433:5432 -d \
+  -e 'PSQL_TRUST_LOCALNET=true' \
+  -e 'PSQL_SLAVE=true' \
+  -e 'REP_HOST=localhost' -e 'REP_PORT=5432' \
+  -e 'REP_USER=replicator' -e 'REP_PASS=replicatorpass' \
+  sameersbn/postgresql:9.4
+```
+
+Create a snapshot downloaded from a master instance:
+```bash
+docker run --name postgresql-slave -p 5433:5432 -d \
+  -e 'PSQL_TRUST_LOCALNET=true' \
+  -e 'PSQL_SLAVE=snapshot' \
```

Comment:
A snapshot is just a backup, it is not maintained through streaming replication but is only a snapshot created when it's booted up.